ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -128,6 +130,14 @@ open class PillButton: UIButton {
         }
     }
 
+    /// Set `backgroundColor` to customize background color of the pill button
+    @objc open var customBackgroundColor: UIColor = Colors.Drawer.background {
```

Comment:
it doesn't seem to make sense the drawer background color is the default color

Reply:
Copied the API from drawer controller. Missed updating the default. Will update in next iteration

---
In reply to: [467155477](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r467155477) [](ancestors = 467155477)

Reply:
It's still using the Drawer background as of your latest iteration.

Reply:
removed now

---
In reply to: [468705802](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468705802) [](ancestors = 468705802)